### ISSUE TITLE
Using Facebook Stetho doesn't allow to inspect SharedPreferenceManager in Android from Brave desktop App

### ISSUE THREAD

**Issue by nawinkhatiwada** [brave/brave-browser#24655]
## Description 
I am using  this version ([Version 1.42.88 Chromium: 104.0.5112.81 (Official Build) (x86_64)](https://brave.com/latest/) ) of brave in MAC OS Monterey and it doesn't allow me to inspect the SharedPreferenceManager of Debuggable app. It only allows me to inspect Sqlite DB infos.

## Steps to Reproduce

   1. Use facebook stetho in android app
             `implementation 'com.facebook.stetho:stetho:1.5.1'`
             `implementation 'com.facebook.stetho:stetho-okhttp3:1.5.1'`
   2. Enable USB debugging in android phone
   3. Open browser and type `brave://inspect/#devices`

## Actual result:
Screenshot Link: 
1. https://drive.google.com/file/d/1IJh3_H9rbWzMFQnME4keQNFaYLsG8_e_/view?usp=sharing
2. https://drive.google.com/file/d/1IsO2C7QAwIDmUjdpxQY7Qi_B6ax5AXlX/view?usp=sharing

## Expected result:
Using `brave://inspect/#devices` must allow me to inspect the SharedPreference of Debuggable app.

## Reproduces how often: 
Easily reproduced

## Brave version (brave://version info)
([Version 1.42.88 Chromium: 104.0.5112.81 (Official Build) (x86_64)](https://brave.com/latest/) )

**Comment by rebron** [comment]
@nawinkhatiwada Does this work with Chrome?

**Comment by nawinkhatiwada** [comment]
@rebron, yes until version `Version 71.0.3578.80 (Official Build) (64-bit)` google chrome supports inspecting both the `SharedPreferenceManager`and `Sqlite-Database` and breaks everything on versions higher than this. As I already mention above, brave browser supports only inspecting `Sqlite-Database` but doesn't support inspecting `SharedPreferences` of the `debuggable` android apps. I have added all the required screenshots which will make the issue more clearer. Please look into this issue with priority. 

**BRAVE**
         Screenshot:
https://drive.google.com/file/d/1we6xMdQbAt2D9hBqToqEtFZKrIfz1jzM/view?usp=sharing
      Version:
[Version 1.43.89 Chromium: 105.0.5195.102 (Official Build) (x86_64)](https://brave.com/latest/)
-> Supports inspection Sqlite DB  only.

**OLD GOOGLE CHROME**
Screenshot:
https://drive.google.com/file/d/115eOIuyO_c9K6Q4HefdilSGP3Xs0aYwn/view?usp=sharing
Version: `Version 71.0.3578.80 (Official Build) (64-bit)`

-> Supports inspection Sqlite DB and SharedPreference Manager Both

**Newer GOOGLE CHROME**
Screenshot:
https://drive.google.com/file/d/1TrS0GL-5zBR6gnUFhmJXCiH3jO4J3Jh6/view?usp=sharing
Version: `Version 105.0.5195.102 (Official Build) (x86_64)`

-> Breaks Everything :rage:

**Comment by nawinkhatiwada** [comment]
@rebron any updates on this?
